README.rst

```diff
@@ -0,0 +1,206 @@
+VulnerableCode
```

Comment:
Thank you for ReSTification.  Why have both `README.md` and  `README.rst` ? Also they both don't look same. I don't care about alignment but please make the GIF work :)

Reply:
I need to remove the README.md.

Reply:
But I also need the gif to be in the repo to make things work.... this https://user-images.githubusercontent.com/28975399/89056138-2c8a8300-d379-11ea-882e-f28f38789cdc.png is not right

Reply:
and this is done now.